web-ng/src/app/shared/converters/firebase-data-converter.ts

```diff
@@ -366,9 +366,10 @@ export class FirebaseDataConverter {
    * @param id the uuid of the project instance.
    * @param data the source data in a dictionary keyed by string.
    */
-  static toFeature(id: string, data: DocumentData): Feature | undefined {
+  static toFeature(id: string, data: DocumentData): Feature | void {
```

Comment:
I've never seen a method return `void` on an error state. Perhaps we should just throw Error instead with a meaningful message?